R/worker_config.R

```diff
@@ -1,9 +1,14 @@
 read_worker_config <- function() {
-  ## Read app config if we have this setup with env vars
-  config <- jsonlite::read_json(system_file("worker_config.json"),
-                                simplifyVector = TRUE)
+  worker_config <- Sys.getenv("HINTR_WORKER_CONFIG")
+  if (!is.null(worker_config) && worker_config != "") {
```

Comment:
rich taught me a trick, i think you can do
```suggestion
  worker_config <- Sys.getenv("HINTR_WORKER_CONFIG", unset = NA)
  if (!is.na(worker_config)) {
```
something like this